Remove '--target=1.15' requirement. (#867)
digital-asset/docs.daml.com: docs/2.10.0/docs/daml/reference/interfaces.rst

Configuration
*************

> In order to use this feature, your Daml project needs to target
> Daml-LF version ``1.15`` or higher which is the current default.

Reviewer: ```suggestion
To use this feature your Daml project must target
Daml-LF version ``1.15`` or higher, which is the current default.
```
